x/gamm/pool-models/balancer/balancerPool.pb.go

```diff
@@ -233,7 +233,7 @@ var xxx_messageInfo_BalancerPool proto.InternalMessageInfo
 func init() {
 	proto.RegisterType((*SmoothWeightChangeParams)(nil), "osmosis.gamm.poolmodels.SmoothWeightChangeParams")
 	proto.RegisterType((*BalancerPoolParams)(nil), "osmosis.gamm.poolmodels.BalancerPoolParams")
-	proto.RegisterType((*BalancerPool)(nil), "osmosis.gamm.poolmodels.BalancerPool")
+	proto.RegisterType((*BalancerPool)(nil), "osmosis.gamm.v1beta1.Pool")
```

Comment:
Lets get the proto package renamed to osmosis.gamm.v1beta1 for everything.

That should 100% remove the problem for SmoothWeightChangeParams. Then this last bit for BalancerPoolParams & BalancerPool vs PoolParams & Pool is this second intense mystery, that we could punt by file renaming. (And asking Regen / cosmos SDK team when we have more time)